CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-1.4.0 / 2021/07/29
+1.4.0 / 2021/08/24
```

Comment:
next release will be 2.0, this needs rebase

Reply:
We decided to release it as 1.4.0 before the TS rewrite

Reply:
master is already v2, so you will need to create a new branch from latest 1.3 and target that instead